kolibri_explore_plugin/collectionviews.py

```diff
@@ -446,7 +452,7 @@ def _remotecontentimport(user, channel_id, node_ids, exclude_node_ids):
             "exclude_node_ids": exclude_node_ids,
         },
     )
-    job_id = remotecontentimport.enqueue(job=job)
+    job_id = remotecontentimport.enqueue(job=job, priority=Priority.HIGH)
```

Comment:
This one … should work and be necessary, I think, but it appears to be a no-op:

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/tasks/registry.py#L262-L273

It ends up using the priority from `register_task`, ignoring the keyword argument we pass. It doesn't look like anything in Kolibri expects this to work the way we are expecting here, so I think we need to either hack at a lower level in the tasks registry code (see the linked function) or add our own task for this instead.

Reply:
OK, it seems you found a way to make it work by passing the default queue. Thanks! Merging.